compiler/serialization/bin.js

```diff
@@ -9,35 +9,36 @@ const {
   convertLogic,
   convertDocument,
   extractProgram,
-} = require('./lib/index')
```

Comment:
pretty sure this was a bug as the `lib` folder isn't included in the npm package

Reply:
oh yeah, that seems likely